src/server/utils/cdnCache.js

```diff
@@ -26,15 +26,18 @@ export const oneAppModuleCachePath = path.join(oneAppDirectoryPath, cacheFileNam
 export const showCacheInfo = async () => {
   try {
     const stats = await fsPromises.stat(oneAppModuleCachePath);
+    if (stats.size > 1024 * 1024 * 10) {
+      console.warn(chalk.bold.redBright('🚨🚨🚨: Cache file size is more than 10MB. 🚨🚨🚨'));
+    }
     const fileSizeOnMB = stats.size / (1024 * 1024); // bytes to mb
     const message = `File size of ${cacheFileName}: ${chalk.bold.greenBright(fileSizeOnMB.toFixed(2), 'MB')}`;
     const separator = '*'.repeat(message.length);
-    console.log(chalk.bold.cyanBright(separator));
-    console.log(chalk.bold.cyanBright('CACHE INFORMATION'));
-    console.log(message);
-    console.log('To clear the cache, please delete this file:');
-    console.log(`    ${chalk.bold.cyanBright(path.join('~', oneAppDirectoryName, cacheFileName))}`);
-    console.log(chalk.bold.cyanBright(separator));
+    console.info(chalk.bold.cyanBright(separator));
+    console.info(chalk.bold.cyanBright('CACHE INFORMATION'));
+    console.info(message);
+    console.info('To clear the cache, please delete this file:');
+    console.info(`    ${chalk.bold.cyanBright(path.join('~', oneAppDirectoryName, cacheFileName))}`);
```

Comment:
Can you replace the template literal with string interpolation? Doing so helps with performance in cases where the log is dropped and does not have to be evaluated. (Same applies to `message` a couple lines above)

Reply:
do you mean replace with string concatenation?

Reply:
Like:
```
console.info('one-app-dev-cdn loading module map from %s', remoteModuleMapUrl);
```

Reply:
I agree with Matt, we could eliminate a lot of this.